### DISCUSSION TITLE
Pods not working after deploying the application

### DISCUSSION THREAD

**Discussion by shashankdhwaniris** [egovernments/Digit-Core#121] [DIGIT Platform]
![digit-core-pods](https://github.com/egovernments/Digit-Core/assets/131361391/b44d1e11-7271-4c21-a480-14abedf533de)
few pods are not working after deploying the application . it seems init container withing the pods are not able to clone the repositories.

**Comment by talele08** [comment]
Please check if the public key corresponding to the private key [(reference)](https://github.com/egovernments/DIGIT-DevOps/blob/c47fae4ab940683da767b77cee463f0891bb1d5a/config-as-code/environments/egov-demo-secrets.yaml#L38) given in env secrets yaml file is added in the github account having access to the particular repo(MDMS repo). If that is fine please cross-check the know-host in the secret yamls file if it the latest one [(reference)](https://github.com/egovernments/DIGIT-DevOps/blob/c47fae4ab940683da767b77cee463f0891bb1d5a/config-as-code/environments/egov-demo-secrets.yaml#L55)

**Comment by shashankdhwaniris** [comment]
I have created new  keys on the server and updated it in  file DIGIT-DevOps/deploy-as-code/helm/environments/quickstart-config-secrets.yaml as well as my github repository, after updating keys i am able to clone git@github.com:egovernments/configs from command but still git-sync container inside pod egov-mdms-service unable to clone the repository
